Cargo.toml

```diff
@@ -414,7 +428,7 @@ env = { optional = true, version = "0.1.0", package = "uu_env", path = "src/uu/e
 expand = { optional = true, version = "0.1.0", package = "uu_expand", path = "src/uu/expand" }
 expr = { optional = true, version = "0.1.0", package = "uu_expr", path = "src/uu/expr" }
 factor = { optional = true, version = "0.1.0", package = "uu_factor", path = "src/uu/factor" }
-false = { optional = true, version = "0.1.0", package = "uu_false", path = "src/uu/false" }
+"false" = { optional = true, version = "0.1.0", package = "uu_false", path = "src/uu/false" }
```

Comment:
Hmm, it worked before right? Are toml keys not guaranteed to be strings? I think they should be according to https://toml.io/en/v1.0.0#keys